# http://docs.ros.org/en/noetic/api/sensor_msgs/html/msg/Imu.html
imu:
  - name: "imu0"
    select:
      id: "in72e"
      fps: 30
    covariance:
      orientation_covariance_diagonal: [0.05, 0.004, 0.003]
      angular_velocity_covariance_diagonal: [0.05, 0.004, 0.003]
      linear_acceleration_covariance_diagonal: [0.05, 0.004, 0.003]

# http://docs.ros.org/en/noetic/api/sensor_msgs/html/msg/NavSatFix.html
# http://docs.ros.org/en/noetic/api/sensor_msgs/html/msg/NavSatStatus.html
gnss:
  - name: "gnss0"
    select:
      id: "0"
      fps: 1
    
# http://docs.ros.org/en/melodic/api/std_msgs/html/msg/String.html
speaker:
  - name: "speaker"
    select:
      id: "0"
      language: "en"
